script/testing/oltpbench/README.md

```diff
@@ -44,30 +52,33 @@ Currently there are 3 types of the OLTP benchmark testing
     --build-type=release
   ```
 
-### Advanced options
+### OLTPBench test options
 For more configurations, there are 2 ways
 - Via config files
-  - You can refer to the [config files](https://github.com/cmu-db/noisepage/blob/master/script/testing/oltpbench/README.md#config-files) section below for more details
+  - You can refer to the [config files](https://github.com/cmu-db/noisepage/blob/master/script/testing/oltpbench#config-files) section below for more details
 - Via command line options
-  - You can use `-h` or `--help` command line option for more details
-  - *In theory, if the same config are conflicting between config files and command line options, the config from __command line option should prevail__*
+comments  - You can use `-h` or `--help` command line option for more details
 
 ## Test workflow
+Preparation steps of the test
 - Start by invoking `run_oltpbench.py`
   - Load the config file passed in and start an instance of the database with any server_args specified in the config
   - Create a test suite for the whole config file and runs the test suite
-  - Create a `TestOLTPBench` object for the test execution
+  - Create a `TestOLTPBench` object for the test execution, since the `TestOLTPBench` is a subclass of the `TestServer` class, it will a list of tasks as the preparation for the actual test, the most important two of which are as follow
```

Comment:
I think there is a word missing after `it will`

Reply:
added `conduct`